### DISCUSSION TITLE
Is first vowel in quiescent incorrect?

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#26] [Q&A] [answered]
According to the Cambridge Dictionary, the US pronunciation of quiescent is with -ai- as first vowel.

https://dictionary.cambridge.org/pronunciation/english/quiescent

However neither of the outlines has -ai- AOEU: KWEU/KWREFPBT, KWAOE/KWREFPBT, KWEU/KWRE/SEPBT, KWAOE/KWRE/SEPBT

Is that a mistake in the dictionary (or in the Oxford dictionary for that matter?)

**Comment by aerickt** [comment]
/aɪ/ (as in "ice") is given the `AOEU` chord. https://lapwing.aerick.ca/Chapter-08.html#aoeu
Perhaps you're confusing /aɪ/ `AOEU` with /eɪ/ `AEU`?

These words are given `AOEU` and would be one of the pronunciations for quiescent listed in the Cambridge, Oxford, and Wiktionary dictionaries if I'm not mistaken:
https://en.wiktionary.org/wiki/Rhymes:English/a%C9%AA

These words are given `AEU`:
https://en.wiktionary.org/wiki/Rhymes:English/e%C9%AA

**Comment by evgenyfadeev** [comment]
looks like there are two correct pronunciations:  both ai and i (in fact in the first link I provided it's spelled in phonetic script with /i/ but clearly pronounced as /ai/ - in the audio sample). This page below gives two options explicitly:

https://www.oed.com/dictionary/quiescent_adj?tl=true

What would you think about finding a linguistic database of english words with phonetic symbols and map it to steno outlines? Has this been done before or the steno dictionaries are hand-crafted?

**Reply by aerickt** [reply, marked as answer]
Oh I see now, the dictionary is missing `KWAOEU/KWRE/SEPBT`. I misread your original post. I had thought you were asking to add `KWAEU/KWRE/SEPBT` style entries.

> What would you think about finding a linguistic database of english words with phonetic symbols and map it to steno outlines? Has this been done before or the steno dictionaries are hand-crafted?

It's been done: https://github.com/AndrewHess/steno-tools/tree/main
I added every entry to the Lapwing dictionary by hand over the course of a few months in early 2022. Autogenerating the dictionary is a very difficult problem for me; my background is not in CS or anything adjacent (I'm only halfway through my undergrad in civil engineering). However, if someone could autogenerate a Lapwing dictionary, I would be happy to use theirs.
